draft-ietf-quic-transport.md

```diff
@@ -424,12 +426,11 @@ The fields in a Regular packet past the Common Header are the following:
   by the sender.  The first packet sent by an endpoint MUST have a packet number
   of 1.
 
-* AEAD Data: A Regular packet's Common Header and fields are authenticated,
-  but not encrypted.  The rest of a Regular packet, starting with the first
-  frame, is both authenticated and encrypted.  Immediately following the header,
-  Regular packets contain AEAD (Authenticated Encryption with Associated Data)
-  data.  After decryption, the plaintext consists of a sequence of
-  frames, as shown below (frames are described in {{frames}}).
+* AEAD Data: The remainder of a Regular packet is both authenticated and
+  encrypted once packet protection keys are available.  {{QUIC-TLS}} describes
```

Comment:
Hmm, so QUIC-TLS actually does describe how to protect/unprotect packets.  We can move that text if you like and make that not-true, but you probably want to move the key schedule as well.  Then we don't have much left in the TLS doc.